frontend/src/scenes/data-management/event-properties/EventPropertyDefinitionsTable.scss

```diff
@@ -0,0 +1,64 @@
+.event-properties-definition-table {
+    margin-top: 1rem;
+
+    .LemonTable__content > table > tbody {
+        td.definition-column-name {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+
+            .definition-column-name-icon {
+                display: flex;
+                align-items: center;
+                justify-content: flex-start;
+                width: 36px;
+
+                svg.taxonomy-icon {
+                    flex-shrink: 0;
+
+                    &.taxonomy-icon-muted {
+                        path {
+                            fill: var(--taxonomy-icon-muted);
+                        }
+                    }
+
+                    &.taxonomy-icon-verified {
+                        height: 28px;
+
+                        path {
+                            fill: var(--success);
+                        }
+                    }
+                }
+            }
+
+            .definition-column-name-content {
+                display: flex;
+                flex-direction: column;
+                position: relative;
+
+                .definition-column-name-content-title {
+                    font-weight: 600;
+                    cursor: pointer;
+
+                    &:before {
+                        content: '';
+                        position: absolute;
+                        top: -5px;
+                        //bottom: -5px;
+                        left: -10px;
+                        right: -50px;
+                        height: 22px;
+                    }
+                }
+
+                .definition-column-name-content-description {
+                    font-style: italic;
+                    font-size: 13px;
+                    color: var(--text-muted);
+                    text-overflow: ellipsis;
+                }
```

Comment:
We generally use italics for placeholders, with actual values printed in regular style. With everything italic right now, it's hard to fish out user-provided descriptions because they blend in:
<img width="327" alt="Screen Shot 2022-03-11 at 15 15 34" src="https://user-images.githubusercontent.com/4550621/157885022-ae9999db-5735-4a81-bd31-a45151d50ae7.png">